tests/wrappers/time_limit_wrappers_test.py

```diff
@@ -81,5 +81,21 @@ def test_time_limit_fork(env: LlvmEnv):
         fkd.close()
 
 
+# @pytest.mark.xfail(strict=True, reason="https://github.com/facebookresearch/CompilerGym/issues/648")
+def test_time_limit(env: LlvmEnv):
+    """Check CycleOverBenchmarks does not break TimeLimit"""
+    env = TimeLimit(env, max_episode_steps=1)
```

Comment:
I would suggest testing with `max_episode_steps=3`, so that you can test that `done` is False for the first two calls to `step()`.